lib/pups/exec_command.rb

```diff
@@ -1,124 +1,128 @@
-require 'timeout'
+# frozen_string_literal: true
 
-class Pups::ExecCommand < Pups::Command
-  attr_reader :commands, :cd
-  attr_accessor :background, :raise_on_fail, :stdin, :stop_signal
+require 'timeout'
+require 'English'
```

Comment:
what is this thing?

Reply:
this is due to the change from the `$?` global to `$CHILD_STATUS`, see: https://github.com/rubocop/rubocop/issues/1747 and https://ruby-doc.org/stdlib-2.7.1/libdoc/English/rdoc/English.html

alternatively I could ignore that rubocop change?

Reply:
I see, TIL. Sure this change is fine.